core/camel-core-model/src/main/java/org/apache/camel/model/rest/RestDefinition.java

```diff
@@ -170,15 +176,17 @@ public void setSecurityDefinitions(RestSecuritiesDefinition securityDefinitions)
         this.securityDefinitions = securityDefinitions;
     }
 
-    public SecurityRequirementsDefinition getSecurityRequirements() {
+    public List<SecurityDefinition> getSecurityRequirements() {
         return securityRequirements;
     }
 
     /**
      * Sets the security requirement(s) for all endpoints.
      */
-    public void setSecurityRequirements(SecurityRequirementsDefinition securityRequirements) {
-        this.securityRequirements = securityRequirements;
+    public void setSecurityRequirements(Collection<SecurityDefinition> securityRequirements) {
```

Comment:
The getter/setter should be same type, eg List

Reply:
This comes from https://github.com/apache/camel/blob/main/core/camel-core-model/src/main/java/org/apache/camel/model/rest/SecurityRequirementsDefinition.java#L87-L91